types/types.go

```diff
@@ -26,3 +28,12 @@ type SingleSupplierEndpoint struct {
 	SupplierAddress string
 	SessionHeader   *sessiontypes.SessionHeader
 }
+
+// CopyToHTTPHeader copies the poktHeader map to the httpHeader map.
```

Comment:
Why not make it a `POKTHTTPResponse.CopyOverHTTPHeaders(writer.Header())`?

Reply:
@Olshansk, we'll have to implement it for both `POKTHTTP{Request/Response}` and it'll be exactly the same. Then, this will make us create the function variant of it, if we want to factor them.

No strong opinion on this. Happy to add it as a method if you think it improves readability.